### ISSUE TITLE
Import certain symbols in a package?

### ISSUE THREAD

**Issue by liuyxpp** [Eben60/ShareAdd.jl#2]
It seems following use case is not supported by ShareAdd:

```julia
using ShareAdd
@usingany ColorSchemes: tab10
```

Is it possible to add such functionality?

**Comment by Eben60** [comment]
Thank you for the feedback! Yes, it is currently not supported. It shouldn't be difficult to implement it - I'd check how to to.

**Comment by Eben60** [comment]
As a workaround, you can first call `make_importable` function, and then just apply any `using` or `import` statements. In the update to v0.2.1 (just pushed) I made this function exported.

```
using ShareAdd
make_importable("Foo")
using Foo: bar
```
This also should work:

```
using ShareAdd
@usingany Foo
using Foo: bar
```

`make_importable` takes `String` or `Vector{String}` as the argument.

**Comment by liuyxpp** [comment]
The first approach feels a little tedious and un-natural.
As in the second approach, does `@usingany Foo` implicitly import all symbols already? In v0.2.0, it seems this is what ShareAdd do. But the essence of `using Foo: bar` is to import `bar` only.

**Comment by Eben60** [comment]
Sure - it's a workaround, as I actually wrote, which I could enable by adding one word to the code. The issue is not yet closed with it.

**Comment by Eben60** [comment]
Implemented in v0.2.2
